css-align-3/content-distribution/place-content-shorthand-002.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Boc Alignment: place-content shorthand - multiple values specified</title>
+      <link rel="author" title="Javier Fernandez" href="mailto:jfernandez@igalia.com" />
+    <link rel="help" href="http://www.w3.org/TR/css3-align/#propdef-place-content" />
+    <meta name="assert" content="Check that setting two values to place-content sets the first one to 'align-content' and the second one to 'justify-content'." />
+    <script src="../../resources/testharness.js"></script>
+    <script src="../../resources/testharnessreport.js"></script>
+    <script src="../resources/alignment-parsing-utils.js"></script>
+  </head>
+  <body>
+      <div id="log"></div>
+      <script>
+          var values1 = ["normal"].concat(position.slice(1,4), distribution.slice(1,3), baseline.slice(1,2));
+          var values2 = position.slice(4).concat(distribution.slice(3), baseline.slice(2));
```

Comment:
Well, I didn't want to try out all the possible combinations, but I did want to use them as first or second parameter, randomly.

Do you think we should check all the possible combinations ?